hca/__init__.py

```diff
@@ -24,3 +24,5 @@ def get_config():
 
 
 logger = logging.getLogger(__name__)
+
+from . import dss, upload  # noqa
```

Comment:
Rather than do the `# noqa` thing, you can move the code in `__init__.py` into a module in the hca package.

Reply:
Good point, I'll try and refactor this.